actions/shared_create_releases/action.yaml

```diff
@@ -60,17 +60,17 @@ runs:
         GITHUB_TOKEN: ${{ inputs.github_token }}
 
     - name: Create Docker Release
-      if: ${{ inputs.is_release == 'true' }}
+      if: ${{ inputs.is_release == 'true' && inputs.publish_platform == 'Docker' }}
       uses: softprops/action-gh-release@v1
       with:
-        name: ${{ inputs.is_release == 'true' && inputs.publish_platform == 'Docker'}}
+        name: v${{ inputs.version_number_input }}
```

Comment:
I'm unclear on the difference in this file between `version_number_input` and `new_version` - do we need to pass two separate version values? Can it be cleaned up a little? (I know this isn't changed in this PR, feel free to ignore this comment - but I thought it worth mentioning.)